### PR TITLE
add Mautic Style guide

### PR DESCRIPTION
**Pull request by Kelvindest**
Not sure why Gitpod isn't showing the link.

Gitpod link: https://gitpod.io/#https://github.com/mautic/user-documentation/pulls/67

<a href="https://gitpod.io/#https://github.com/mautic/user-documentation/pull/67"></a>

### CODE REVIEW

**Comment by fakela** [comment]
@Kelvindest thanks you did a wonderful job all vale checks passed 👍

**Comment by fakela** [comment]
There is something missing links, i'll open a separate issue for that but this can be merged now that it is ready as it is a source of entry for new contributors

File: docs/contributing/mautic_style_guide.rst [hunk 1]
```diff
@@ -1,2 +1,187 @@
...
+
+Vale
+****
+Vale is a natural language linter that supports plain text, markup (Markdown, reStructuredText, AsciiDoc, and HTML), and source code comments. Vale doesn't attempt to offer a one-size-fits-all collection of rules—instead, it strives to make customization as easy as possible.
+
+How to install vale
+===================
+Before pushing, run Vale and address suggestions and errors as applicable.
+
+#. Install vale
```
Comment (fakela) [inline docs/contributing/mautic_style_guide.rst:13]: You need to add this link https://docs.errata.ai/vale/install to Vale, so its easier for users

File: docs/contributing/mautic_style_guide.rst [hunk 2]
```diff
@@ -1,2 +1,187 @@
...
+   .. image:: images/primary-company.png
+   :width: 600
+   :alt: primary-company
+
+Output:
+
+.. image:: images/primary-company.png
+  :width: 600
+  :alt: primary-company
+
```
Comment (fakela) [inline docs/contributing/mautic_style_guide.rst:187]: We need to add a section that explains how we link internal and external pages

**Review by fakela — CHANGES_REQUESTED** [review]

**Comment by RCheesley** [comment]
@fakela I have pushed some commits to address the feedback you left in the review - just double checking it renders OK on Gitpod.

**Comment by fakela** [comment]
> Ready for review - the build errors are the links we are already aware of.

Alright it's good to go 🎉

**Review by fakela — APPROVED** [review]

**Pull request merged**